src/notation/imasternotation.h

```diff
@@ -50,7 +50,7 @@ class IMasterNotation
     virtual const ExcerptNotationList& potentialExcerpts() const = 0;
 
     virtual void initExcerpts(const ExcerptNotationList& excerpts) = 0;
-    virtual void addExcerpts(const ExcerptNotationList& excerpts) = 0;
+    virtual void addExcerpts(const ExcerptNotationList& excerpts, bool moveToEndIfExists = true) = 0;
```

Comment:
where are you setting the different values for moveToEndIfExists?

Reply:
It's here to make the behavior of this function clear